beacon-chain/sync/service.go

```diff
@@ -84,6 +85,7 @@ func NewRegularSync(cfg *Config) *Service {
 		stateSummaryCache:    cfg.StateSummaryCache,
 		stateGen:             cfg.StateGen,
 		blocksRateLimiter:    leakybucket.NewCollector(allowedBlocksPerSecond, allowedBlocksBurst, false /* deleteEmptyBuckets */),
+		chainStarted:         roughtime.Now().After(cfg.Chain.GenesisTime()),
```

Comment:
Not sure this is correct, wouldn't this always be true ? 
```
cfg.chain.GenesisTime()
```
wouldn't this be the zero value of `time.Time`